tensorflow_estimator/python/estimator/early_stopping.py

```diff
@@ -130,6 +131,8 @@ def stop_if_higher_hook(estimator,
       default, `estimator.eval_dir()` will be used.
     min_steps: `int`, stop is never requested if global step is less than this
       value. Defaults to 0.
+    min_delta: `float`, minimum required change in the given metric to determine
```

Comment:
Is it accepted to pass in negative `min_delta`?

Reply:
No it is not, I can enforce this by grabbing the absolute value or asserting > 0.

Reply:
How does it sound if we raise `ValueError` if `min_delta < 0` and add this info to the docstring?

Reply:
Sounds good, I added the validation to the `stop_if_higher_hook` function.